video/out/vo_gpu_next.c

```diff
@@ -1001,8 +1005,8 @@ static void draw_frame(struct vo *vo, struct vo_frame *frame)
         pl_swapchain_colorspace_hint(p->sw, NULL);
```

Comment:
Make sure libplacebo doesn't duplicate our job. guard `pl_swapchain_colorspace_hint` with `if (!pass_colorspace)` below. It should be enough for now. We might need to ensure that swapchain format is not set to 8-bits, when we try to pass colorspace, this what libplacebo would do internally.